src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -15,6 +17,7 @@ public class UserRequestDto {
     @Size(min = 8, max = 40)
     private String password;
     private String repeatPassword;
+    private Set<Role> roles;
```

Comment:
In a DTO, it's better to avoid using entity classes directly. Instead of `Set<Role>`, use `Set<String>` or create a RoleDto class.